app/constants/image.js

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export const DEFAULT_MIME_TYPE = 'image/png';
```

Comment:
0/5 on this, but we could probably leave this constant in `image_cache_manager.js` to avoid creating a new file just for this.